app/pages/dev-classifier/mock-data.coffee

```diff
@@ -518,20 +518,33 @@ workflow = apiClient.type('workflows').create
     confusionsOrder: []
     confusions: {}
 
+labs1715QueryPresent = ->
+  urlParams = new URLSearchParams(window.location.search);
+  labs1715QueryPresent = urlParams.get('1715labs');
+  console.log(labs1715QueryPresent);
+  if labs1715QueryPresent then true else false
+
 subject = apiClient.type('subjects').create
   id: 'MOCK_SUBJECT_FOR_CLASSIFIER'
   # Images originally from lorempixel.com shared under CC BY-SA,
   # but the service is often slow and/or fails to load at all.
   # Noted original source next to each.
   locations: if navigator?.onLine
-    [
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/landscape.jpeg"} # //lorempixel.com/900/600/animals/1
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/portrait.jpeg"} # //lorempixel.com/600/900/animals/2
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/very-wide.jpeg"} # //lorempixel.com/1900/1000/animals/3
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/very-tall.jpeg"} # //lorempixel.com/1000/1900/animals/4
-      {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/small.jpeg"} # //lorempixel.com/400/300/animals/4
-      {'text/plain': "https://static.zooniverse.org/preview.zooniverse.org/panoptes-front-end/highlighter-task/assets/dev-classifier/algernon.txt"}
-    ]
+    if labs1715QueryPresent()
+      [
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/london_street_view.jpeg"}
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/greenwich_satellite_image.jpeg"}
+        {'image/jpeg': "#{window.location.origin}/assets/dev-classifier/1715_labs/building_plans.jpeg"}
+      ]
```

Comment:
Without the `tex/plain` subject, the dummy text task might fail.

Reply:
nice catch - thanks.

Reply:
It doesn't work, but it doesn't crash the classifier 🎉

Reply:
👍 I've been thinking the text task should automatically bring the text frame into view, for convenience. Maybe by adding a new static property for tasks, which is the default file type that those tasks act on.